### ISSUE TITLE
React 15 warning for key prop

### ISSUE THREAD

**Issue by rgbkrk** [rexxars/commonmark-react-renderer#16]
```
Warning: Element: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)
```

Everything else is fine for React 15, though the use of key as generated here should have a different name.

**Comment by rgbkrk** [comment]
Started on a patch for this, will have to try again next week unless someone else gets to it.

**Comment by perrin4869** [comment]
+1

**Comment by rexxars** [comment]
Fixed in 4.1.3. Thanks for the interest!
